src/core/etl/tests/Flow/ETL/Tests/Integration/DataFrame/TransformTest.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Integration\DataFrame;
+
+use function Flow\ETL\DSL\{df, from_array, lit, ref, with_entry};
+use Flow\ETL\Tests\FlowTestCase;
+
+final class TransformTest extends FlowTestCase
+{
+    public function test_transform_with_entries() : void
+    {
+        $rows = df()
+            ->read(from_array([
+                ['id' => 1, 'status' => 'active', 'amount' => 41_000],
+                ['id' => 2, 'status' => 'inactive', 'amount' => 250],
+                ['id' => 3, 'status' => 'active', 'amount' => 1_000],
+            ]))
+            ->transform(with_entry('amount', ref('amount')->divide(100)))
+            ->transform(with_entry('currency', lit('PLN')))
+            ->fetch()
+            ->toArray();
+
+        self::assertEqualsWithDelta(
+            [
+                ['id' => 1, 'status' => 'active', 'amount' => 410.00, 'currency' => 'PLN'],
+                ['id' => 2, 'status' => 'inactive', 'amount' => 2.5, 'currency' => 'PLN'],
+                ['id' => 3, 'status' => 'active', 'amount' => 10.00, 'currency' => 'PLN'],
+            ],
+            $rows,
+            0.01
```

Comment:
why is this needed?

Reply:
great catch! 
It's not, I think I copied it from another place that was actually failing due to floating point comparison